java/code/src/com/redhat/rhn/common/db/datasource/xml/ChannelReport_queries.xml

```diff
@@ -44,14 +42,38 @@
                       , rhnerrata.advisory_type
                       , rhnerrata.issue_date
                       , rhnerrata.update_date
-                      , rhnerrata.product
-                      , rhncve.name AS cve
+                      , CASE
+                          WHEN rhnerrataseverity.rank = 0 then 'Critical'
+                          WHEN rhnerrataseverity.rank = 1 then 'Important'
+                          WHEN rhnerrataseverity.rank = 2 then 'Moderate'
+                          WHEN rhnerrataseverity.rank = 3 then 'Low'
+                          ELSE NULL
+                        END AS severity
+                      , CASE
+                          WHEN rb.keyword IS NOT NULL THEN 'Y'
+                          ELSE 'N'
+                        END AS reboot_required
+                      , CASE
+                          WHEN rs.keyword IS NOT NULL THEN 'Y'
+                          ELSE 'N'
+                        END AS affects_package_manager
+                      , CASE
+                          WHEN er.keyword IS NOT NULL THEN 'Y'
+                          ELSE 'N'
+                        END AS retracted
```

Comment:
what is more convenient these days? 
`Y/N` values or real BOOLEANs?

Reply:
Changed the query and the table to use booleans